website/content/en/docs/faqs/_index.md

```diff
@@ -20,6 +20,12 @@ For further context about the relationship between Kubebuilder and Operator SDK,
 
 Please see the upstream [Controller Runtime FAQ][cr-faq] first for any questions related to runtime mechanics or controller-runtime APIs.
 
+## Can I customize the projects generated with SDK tool?
+
+After using the CLI to create your project, you are free to customise how you see fit. Bear in mind, that it is not recommended to deviate from the proposed layout unless you know what you are doing.
```

Comment:
```suggestion
After using the CLI to create your project, you are welcome to customize based on how you see fit. Also please note that it is not recommended to deviate from the proposed layout unless you are fully aware of the repercussions of what you are doing.
```

Reply:
In this case, I'd like to keep the original text. The reason for that is because it is more directly and makes it more clear what we are trying to share. Note that it is very common to find in k8s/controller runtime docs `unless you know what you are doing.`

Reply:
Sounds good. Can you keep the text and fix the typo:
"After using the CLI to create your project, you are free to customize based on how you see fit. Please note that it is not recommended to deviate from the proposed layout unless you know what you are doing."